lib/src/world.cairo

```diff
@@ -58,6 +62,8 @@ mod World {
         partition_len: LegacyMap::<(felt252, felt252), usize>,
         partition: LegacyMap::<(felt252, felt252, felt252), felt252>,
         module_registry: LegacyMap::<starknet::ContractAddress, bool>,
+        role_admin: LegacyMap::<felt252, felt252>,
```

Comment:
Since Cairo 1 has types now, it would be better if this was `LegacyMap::<felt252, ContractAddress>`

Reply:
@milancermak the role admin maps from a role to the role administrating this role, not to a contract address. Based off the [openzeppelin docs](https://docs.openzeppelin.com/contracts-cairo/0.4.0/access#granting_and_revoking_roles).

Reply:
Huh, I misunderstood, I thought the admin of a role is an account, not another role. All good then.